.github/workflows/ci.yml

```diff
@@ -85,14 +85,14 @@ jobs:
   node_with_go_waku_master:
     uses: ./.github/workflows/test-node.yml
     with:
-      nim_wakunode_image: wakuorg/go-waku:latest
+      nim_wakunode_image: harbor.status.im/wakuorg/go-waku:latest
```

Comment:
harbor.status.im is the official repo for the nwaku and gowaku images and from there they are replicated on docker hub
But the replication had issues in the past and docker hub images were outdated.
So for the interop testing project I was instructed to use the images from harbor.status.im because it's more likely for them to be up to date.
That's why I updated them on js-waku side as well